### PR TITLE
Minor: Refactor benchmark imports to use `util` module

### PR DESCRIPTION
**Pull request by loloxwg**
This pull request focuses on refactoring the import statements across multiple files in the `benchmarks` module to improve code organization by consolidating utility imports into a single module.

## Which issue does this PR close?

Closes #.

## Are these changes tested?
yes

### CODE REVIEW

File: benchmarks/src/bin/h2o.rs [hunk 1]
```diff
@@ -26,7 +26,7 @@ use datafusion::datasource::listing::{
 use datafusion::datasource::MemTable;
 use datafusion::prelude::CsvReadOptions;
 use datafusion::{arrow::util::pretty, error::Result, prelude::SessionContext};
-use datafusion_benchmarks::BenchmarkRun;
+use datafusion_benchmarks::util::BenchmarkRun;
```
Comment (findepi) [inline benchmarks/src/bin/h2o.rs:29]: i think the real problem (if any) is that `datafusion_benchmarks::BenchmarkRun` and `datafusion_benchmarks::util::BenchmarkRun` are both valid import path.

there are 3 ways to go about it

- stop re-exporting BenchmarkRun from crate root, require `::util::`
- make util private, accept only BenchmarkRun import from crate root
- ignore the problem and give up on consistency

File: benchmarks/src/lib.rs [hunk 2]
```diff
@@ -21,5 +21,4 @@ pub mod imdb;
 pub mod parquet_filter;
 pub mod sort;
 pub mod tpch;
-mod util;
-pub use util::*;
+pub mod util;
```
Comment (alamb) [inline benchmarks/src/lib.rs:24]: this seems like an improvment to me (not re-export everything from `util` into the `banchmark` crate namespace

File: benchmarks/src/bin/h2o.rs (hunk 1 above)
Comment (alamb) [reply benchmarks/src/bin/h2o.rs:29]: > stop re-exporting BenchmarkRun from crate root, require ::util::

I think that is what this PR does which seems reasonable to me

**Review by alamb — APPROVED** [review]
Thank you @loloxwg  and @findepi for the review

**Pull request merged**